onedal/neighbors/tests/test_knn_classification.py

```diff
@@ -0,0 +1,152 @@
+#===============================================================================
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+import pytest
+import numpy as np
+from numpy.testing import assert_array_equal, assert_array_almost_equal
+
+from onedal.neighbors import KNeighborsClassifier
+from onedal.tests.utils._device_selection import (get_queues,
+                                                  pass_if_not_implemented_for_gpu)
+
+from sklearn.utils.estimator_checks import check_estimator
+import sklearn.utils.estimator_checks
+from sklearn import datasets
+from sklearn.metrics.pairwise import rbf_kernel
+from sklearn.datasets import make_blobs
+from sklearn.model_selection import train_test_split
+
+
+N_NEIGBORS = 2

```

Comment:
It might be made as a parameter to the tests but not the global variable

Reply:
removed